src/inputs/dropzone.jsx

```diff
@@ -244,7 +245,7 @@ class MyDropzone extends React.Component {
                 </p>
                 {accept && (
                   <p className='u-text-center'>
-                    Only ${accept} type files accepted.
+                    Only {accept} type files accepted.
```

Comment:
Does this need double curly braces `{{accept}}`?

Reply:
We want the file types to render inline, so nope.